src/encoded/static/components/search.js

```diff
@@ -656,6 +656,22 @@ const Term = (props) => {
         negationHref = `${searchBase}${field}!=${globals.encodedURIComponent(term)}`;
     }
 
+    if (facet.appended === 'true') {
+        const isNegated = facet.isEqual === 'false';
+        const vfield = filters.find(f => f.field === facet.field);
+        const vhref = vfield ? vfield.remove : '';
+        return (
+            <li className={`facet-term${isNegated ? ' negated-selected' : (selected ? ' selected' : '')}`}>
+                {statusFacet ? <Status item={term} badgeSize="small" css="facet-term__status" noLabel /> : null}
+                <a className="facet-term__item" href={vhref} onClick={href ? onFilter : null}>
+                    <div className="facet-term__text">
+                        {em ? <em>{title}</em> : <span>{title}</span>}
+                    </div>
+                </a>
+            </li>
+        );
+    }
+
```

Comment:
I think this is good, but this return value is a near duplicate of the first several lines of the following return value. Do you think there’s a way to reduce this duplication, or would that make things _more_ complicated and difficult to maintain? I’ll go with your judgement here. This duplication looks like a maintenance hassle, but that’s OK if the alternative is worse.

Reply:
I went with the option of having two separate code; that can enable them to change independently.